### PR TITLE
Starliner Boring Crew Services Configs

### PR DESCRIPTION
**Pull request by SkyPhoenix999**
New BCS Configs

- All Starliner Parts
- RS-88 Config Update

### CODE REVIEW

File: GameData/RealismOverhaul/RO_SuggestedMods/BoringCrewServices/RO_Starliner.cfg [hunk 1]
```diff
@@ -0,0 +1,780 @@
...
+		}
+		TANK
+		{
+			name = WasteWater
+			amount = 0
+			maxAmount = 34.47
+		}
+	}
+}
+@PART[BCS_Centauri_CrewCapsule]:NEEDS[ProfileRealismOverhaul]:AFTER[RealismOverhaul]
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/BoringCrewServices/RO_Starliner.cfg:180]: This should be in the kerbalism profile, not RO.

File: GameData/RealismOverhaul/Engine_Configs/RS88_Config.cfg [hunk 2]
```diff
@@ -36,29 +36,35 @@
 //	=================================================================================
 
 //	Sources:
-//		[1]NTRS - RS-88 Pad Abort Demonstrator Thrust Chamber Assembly:					http://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/20050237017.pdf
-//		[2]Encyclopedia Astronautica - RS-88 engine:										http://astronautix.com/r/rs-88.html
-//		[3]Boeing - Design Considerations for a Commercial Crew Transportation System:		http://www.boeing.com/assets/pdf/defense-space/space/ccts/docs/Space_2011_Boeing.pdf
-//		[4]https://arc.aiaa.org/doi/epdf/10.2514/6.2005-4422
-//		[5]https://www.spaceflightinsider.com/organizations/boeing/launch-abort-engines-boeing-cst-100-starliner-undergo-testing/
-//		[6]https://www.americaspace.com/2014/08/08/aerojet-rocketdyne-completes-ccicap-work-with-boeings-cst-100-spacecraft/
+
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/Engine_Configs/RS88_Config.cfg:39]: Could you try to avoid messing up the references here?

**Review by Capkirk123 — CHANGES_REQUESTED** [review]

File: GameData/RealismOverhaul/RO_SuggestedMods/BoringCrewServices/RO_Starliner.cfg [hunk 3]
```diff
@@ -0,0 +1,780 @@
...
+	{
+		name = ModuleRealAntenna
+		referenceGain = 1.5
+		RFBand = S		//Default to S-band
+	}
+	MODULE
+	{
+		name = ModuleRealAntenna
+		antennaDiameter = 0.8	//Same as Orion because I cannot find any data
+		RFBand = K	//Default to K-band
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/BoringCrewServices/RO_Starliner.cfg:131]: Shpi;d be Ka-band

File: GameData/RealismOverhaul/RO_SuggestedMods/BoringCrewServices/RO_Starliner.cfg [hunk 4]
```diff
@@ -0,0 +1,780 @@
...
+
+//	Docking Cover
+@PART[BCS_Centauri_NDS_cover]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.6
+	@title = CST-100 Docking Port Cover
+	@manufacturer = #roMfrBoeing
+	@description = A hingable cover for the NASA Docking System port on the front end of Boeing's Starliner capsule, designed to shield the docking port when not attached to the Space Station.
+	%skinTempTag = HRSI
```
Comment (Capkirk123) [inline GameData/RealismOverhaul/RO_SuggestedMods/BoringCrewServices/RO_Starliner.cfg:388]: Do these parts want or need to be HRSI?

Comment (SkyPhoenix999) [reply GameData/RealismOverhaul/RO_SuggestedMods/BoringCrewServices/RO_Starliner.cfg:388]: Part is made of and covered with the same insulation material as the rest of the capsule.

It doesn’t _need_ to be HRSI to survive re-entry but it logically makes sense.

Comment (Capkirk123) [reply GameData/RealismOverhaul/RO_SuggestedMods/BoringCrewServices/RO_Starliner.cfg:388]: This article is pretty old (2013) but claims that the CST-100 backshell TPS is a mixture of BRI-18 (HRSI) and FRSI blankets. Looking at actual pictures of Starliner, it's pretty obvious that the white parts of the backshell are FRSI blankets, the only HRSI is the black tiles around the base of the heat shield and hatches,

**Comment by SkyPhoenix999** [comment]
Corrections Made for the configs

Kerbalism Config moved to a pull on the ROKerbalism Page

Found Here: https://github.com/KSP-RO/ROKerbalism/pull/164

File: GameData/RealismOverhaul/RO_SuggestedMods/BoringCrewServices/RO_Starliner.cfg (hunk 1 above)
Comment (SkyPhoenix999) [reply GameData/RealismOverhaul/RO_SuggestedMods/BoringCrewServices/RO_Starliner.cfg]: Moved to ROKerbalism

**Review by Capkirk123 — APPROVED** [review]

**Pull request merged**